apps/codeforafrica/src/lib/api.ghost.js

```diff
@@ -0,0 +1,71 @@
+import GhostContentAPI from "@tryghost/content-api";
+
+const GHOST_API_URL = process.env.GHOST_CMS_URL;
+const GHOST_API_KEY = process.env.GHOST_CMS_API_KEY;
+
+const api = new GhostContentAPI({
+  url: GHOST_API_URL,
+  key: GHOST_API_KEY,
+  version: "v3.0",
+});
+
+export async function getAllPosts(options) {
+  const { limit = 10, page = 1, ...other } = options || {};
+
+  const posts = await api.posts.browse({
+    limit,
+    page,
+    fields: "id,title,slug,published_at,feature_image,excerpt",
+    include: "authors,tags",
+    ...other,
+  });
+  return posts;
+}
+
+export async function getPost(slug) {
+  const posts = api.posts.read({
+    slug,
+    fields:
+      "id,title,slug,published_at,feature_image,excerpt,custom_excerpt,excerpt,html",
+    include: "authors",
+  });
+  return posts;
+}
+
+export async function getAllTags(options) {
```

Comment:
@kilemensi Any idea of how to filter the tags based on another tag? For example, we could want to only get tags for stories alone.

Reply:
No I don't have any @kelvinkipruto ... Lets just pull all posts by `primary_tag` and then concat tags from each post either manually or using [`helpers`](https://ghost.org/docs/content-api/javascript/#helpers).